usort/api.py

```diff
@@ -49,6 +50,23 @@ def usort_bytes(data: bytes, config: Config, path: Optional[Path] = None) -> Res
         )
 
 
+def usort_bytes(
+    data: bytes, config: Config, path: Optional[Path] = None
+) -> Tuple[bytes, str]:
+    """
+    Returns (new_bytes, encoding_str) after sorting.
+
+    DEPRECATED: use `usort()` directly instead.
+    """
+    warn("use usort() instead", DeprecationWarning)
+
+    result = usort(data=data, config=config, path=path)
+    if result.error:
+        raise result.error
+
+    return result.output, result.encoding or ""
```

Comment:
The `or ""` is going to get the caller in trouble, I'd rather assert here.  LibCST should use 'utf-8' when unknown anyway.